packages/dear-imgui/cpp/app/src/xframes.cpp

```
@@ -474,26 +474,26 @@ void ReactImgui::RenderDebugWindow() {
 }
 
 template <typename T>
-void ReactImgui::ExtractNumberFromStyleDef(const json& styleDef, const std::string key, T& value) {
+void XFrames::ExtractNumberFromStyleDef(const json& styleDef, const std::string key, T& value) {
```

cppcheck

[performance] Function parameter 'key' should be passed by const reference.

(passedByValue)